skate-plugin/src/main/resources/messages/skateBundle.properties

```
@@ -4,3 +4,8 @@ skate.configuration.choosePath.dialog.title=Choose Changelog File
 skate.configuration.enableWhatsNew.title=Enable What's New Panel
 skate.configuration.enableWhatsNew.description=If enabled, shows the What's New Panel if there are new changes detected in a specified changelog file.
 skate.modelTranslator.description=Generate translator body
+skate.configuration.enableFeatureFlagLinking.title=Enable Feature Flag Linking
+skate.configuration.enableFeatureFlagLinking.description=If enabled, feature flag URL will be shown in the quick documentation tooltip.
```

This isn't quite right

```suggestion
skate.configuration.enableFeatureFlagLinking.description=If enabled, show inspections for opening feature flags.
```